### PR TITLE
FL: add bodycam footage released by Fort Lauderdale PD

### PR DESCRIPTION
**Pull request by sprucelee**

### CODE REVIEW

File: reports/Florida.md [hunk 1]
```diff
@@ -35,6 +35,20 @@ id: fl-fortlauderdale-2
 * [CNN Interview](https://twitter.com/greg_doucette/status/1270479109982912514)
 
 
+### Police laugh and joke while "indiscriminately firing" rubber bullets and tear gas | May 31st
```
Comment (emarcey) [inline reports/Florida.md:38]: This title needs a little work. First, "indiscriminately firing" is not neutral. Second, it's a nitpick, but it seems that this dialogue happened after firing. It's a small thing but I think there's a difference in degree between while and after

File: reports/Florida.md [hunk 2]
```diff
@@ -35,6 +35,20 @@ id: fl-fortlauderdale-2
...
+
+Bodycam video released by Fort Lauderdale PD on Youtube shows police laughing and joking when they believe their cams are in "standby" mode and not recording. At 2:55, a cop can be heard saying "if you need it, give me a target": the Miami Herald quotes a retired Lieutenant Colonel as saying "If you're not seeing a threat, why are you asking for a threat? That would almost lead you to believe that they were indiscriminately firing."
```
Comment (emarcey) [inline reports/Florida.md:40]: I don't think a colon is the punctuation you want at the end of the first quote

File: reports/Florida.md [hunk 3]
```diff
@@ -35,6 +35,20 @@ id: fl-fortlauderdale-2
...
+
+tags: shoot, rubber-bullets, tear-gas
```
Comment (emarcey) [inline reports/Florida.md:42]: I think abuse-of-power, body-cam, and incitement are useful tags to add here as well.

File: reports/Florida.md [hunk 4]
```diff
@@ -35,6 +35,20 @@ id: fl-fortlauderdale-2
...
+
+id: fl-fortlauderdale-3
+
+**Links**
+
+* [Bodycam video released by FLPD](https://www.youtube.com/watch?v=JJx9NgTe8Ns)
+* https://www.miamiherald.com/news/local/community/broward/article243932047.html
```
Comment (emarcey) [inline reports/Florida.md:49]: Can you add more context from this article into the description. In particular, the pieces about the body cams and the various response from different government officials

File: reports/Florida.md (hunk 1 above)
Comment (sprucelee) [reply reports/Florida.md]: Changed the title to focus on the body cams

File: reports/Florida.md [hunk 5]
```diff
@@ -35,6 +35,20 @@ id: fl-fortlauderdale-2
 * [CNN Interview](https://twitter.com/greg_doucette/status/1270479109982912514)
 
 
+### Police laugh and joke when they think their body cams are off | May 31st
+
+Body cam video released by Fort Lauderdale PD on Youtube shows police laughing and joking when they believe their cams are in "standby" mode and not recording. Policing experts who reviewed the footage say that the behavior is unprofessional and that the officers felt "comfortable about talking about turning the camera off." At 2:55, a cop can be heard saying "if you need it, give me a target." The Miami Herald quotes a retired Lieutenant Colonel as commenting, "If you're not seeing a threat, why are you asking for a threat? That would almost lead you to believe that they were indiscriminately firing."
```
Comment (emarcey) [inline reports/Florida.md:40]: This looks pretty good, I would also just add back in that this is after having shot rubber bullets/ tear gas

Comment (sprucelee) [reply reports/Florida.md]: I've added this in. I split the first sentence into two for readability.

**Pull request merged**